test/e2e/self_hosted.go

```diff
@@ -389,13 +389,15 @@ func SelfHostedSpec(ctx context.Context, inputGetter func() SelfHostedSpecInput)
 	})
 
 	AfterEach(func() {
+		//dumpNodesAndPods(ctx, specName, input.BootstrapClusterProxy, input.ArtifactFolder, namespace, cancelWatches, clusterResources.Cluster, input.E2EConfig.GetIntervals, input.SkipCleanup)
 		if selfHostedNamespace != nil {
 			// Dump all Cluster API related resources to artifacts before pivoting back.
 			framework.DumpAllResources(ctx, framework.DumpAllResourcesInput{
 				Lister:    selfHostedClusterProxy.GetClient(),
 				Namespace: namespace.Name,
 				LogPath:   filepath.Join(input.ArtifactFolder, "clusters", clusterResources.Cluster.Name, "resources"),
 			})
+			dumpNodesAndPods(ctx, specName, selfHostedClusterProxy, input.ArtifactFolder, namespace, cancelWatches, clusterResources.Cluster, input.E2EConfig.GetIntervals, input.SkipCleanup)
```

Comment:
Let's drop this as it's the same as what's inside `dumpNodesAndPods`

Reply:
added via 7d6e324612b0a66396b259ef872fb0b8c47b19c9